lxd/device/disk.go

```diff
@@ -1734,7 +1734,13 @@ func (d *disk) getParentBlocks(path string) ([]string, error) {
 		// Accessible btrfs filesystems
 		output, err := shared.RunCommand("btrfs", "filesystem", "show", dev[1])
```

Comment:
@stgraber I would like to add a comment here explaining why we attempt to use `btrfs filesystem show` in the first place, but I do not know why this is, could you enlighten me and I'll update the comment. Thanks :)